apps/web/pages/booking/[uid].tsx

```diff
@@ -197,12 +199,10 @@ export default function Success(props: SuccessProps) {
     formerTime,
     email,
     seatReferenceUid,
+    tz: timeZoneQuery,
   } = querySchema.parse(router.query);
 
-  const tz =
-    (isSuccessBookingPage
-      ? props.bookingInfo.attendees.find((attendee) => attendee.email === email)?.timeZone
-      : props.bookingInfo.eventType?.timeZone || props.bookingInfo.user?.timeZone) || timeZone();
+  const tz = timeZoneQuery ? timeZoneQuery : props.tz ? props.tz : timeZone();
```

Comment:
if `tz` query param present use that or if user is authenticated user that or fallback to guessing his timezone